Gemfile

```diff
@@ -5,14 +5,17 @@ source 'https://rubygems.org'
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gem 'byebug'
-gem 'config', '~> 3.1'
-gem 'parallel', '~> 1.21'
-gem 'pastel', '~> 0.8'
-gem 'rubocop', '~> 1.24'
-gem 'thor', '~> 1.1'
-gem 'tty-progressbar', '~> 0.18'
-gem 'tty-prompt', '~> 0.23'
-gem 'tty-table', '~> 0.12'
+gem 'config'
+gem 'parallel'
+gem 'pastel'
+gem 'rubocop'
+gem 'thor'
+gem 'tty-file'
+gem 'tty-logger'
+gem 'tty-markdown'
```

Comment:
These are the three new dependencies, all from the `TTY` family of libraries we were already using here.